### PR TITLE
Add and Enhance "Thank you for contributing" with a nice JS effect in another language

### PR DESCRIPTION
**Pull request by NishithP2004**
- [ ] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](url)

- Fixes #<Issue Number>

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by simarsingh24** [comment]
please make the requested changes @NishithP2004

File: img/students/Nishith.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: Nishith
+  github: NishithP2004
+  image: DEE244C4-C7AC-45A4-9FB3-82B6EA8EA512.png
```
Comment (abishekvashok) [inline img/students/Nishith.yml]: Try to name an image logically. See the readme.

**Comment by Ryan10145** [comment]
@NishithP2004 The changes to the Thank You section should be done in thankyou.js. You shouldn't make a new file called "Thank you" as it doesn't change the Thank You section at all.

Additionally, the changes to add yourself as a student should be done in the students.yml file, not in a new file you created. 

However, as others have stated, you should limit your PR to only one change, so you should separate your changes into different pull requests.

**Comment by fragm3** [comment]
Closing this, duplicate of https://github.com/fossasia/gci18.fossasia.org/pull/441

**Pull request closed without merging**